io/zenoh-links/zenoh-link-tls/src/utils.rs

```diff
@@ -188,6 +188,11 @@ impl TlsServerConfig {
             bail!("No private key found for TLS server.");
         }
 
+        // Install rustls provider
+        rustls::crypto::ring::default_provider()
+            .install_default()
+            .ok();
```

Comment:
I think it's better to return an error if this operation fails.
```suggestion
            .map_err(|err| zerror!("Error installing TLS crypto provider: {err}."))?;
```